apps/sms/style/sms.css

```diff
@@ -464,7 +463,7 @@ section[role="region"].new > header:first-child form {
   color: #008787;
 }
 
-.has-carrier .carrier-wrapper {
+.carrier-wrapper, .has-carrier {
   display: block;
```

Comment:
I think you missed the goal of this rule.

There are cases where we want to hide it and we want to display it only in some cases.

Here are the cases where it should be hidden: group MMS, new message panel, select mode (maybe I forget some)